src/skip-to-content/skip-to-content.scss

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+
+@mixin iui-skip-to-content {
+  @include iui-blur;
+  @include iui-focus($offset: 2px, $thickness: 2px);
+
+  border-radius: 99999px;
```

Comment:
I think we might want to save this border-radius as a global variable somewhere, we need it quite often

Reply:
I can add this as a global variable. Where else are similar values used? I couldn't find any (in the CSS project) when I looked.

Reply:
This PR has a better chance of being approved before #465.  @elephantcatdog if you want to add a variable in `/src/style/variable.scss` for `99999px` I'll use that in the toggle switch PR after this PR gets approved.

Reply:
I added $iui-border-radius-huge but it is 9999px currently instead of 99999px.

Reply:
Maybe it should be called `$iui-border-radius-round` or `$iui-border-radius-pill`. I see tailwind calls it `rounded-full` https://tailwindcss.com/docs/border-radius

Reply:
`$iui-border-radius-round` makes the most sense to me.

Reply:
Fixed